config/esp32/.gn

```diff
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import("//build_overrides/chip.gni")
+
 # The location of the build configuration file.
-buildconfig = "//build/config/BUILDCONFIG.gn"
+buildconfig = rebase_path("${chip_root}/build/config/BUILDCONFIG.gn")
```

Comment:
I'm not sure if it matters, but maybe chip_root should be an absolute path already in which case rebase_path here wouldn't be needed (worth considering).

Reply:
This doesn't work for some reason. The buildconfig seems to ignore "../" strings and so, we need to pass the absolute path. Any ideas here?

Reply:
I meant that we might assign chip_root = rebase_path("//../../..") and use "${chip_root}/build/config/BUILDCONFIG.gn" here.

Reply:
Ok will try this.